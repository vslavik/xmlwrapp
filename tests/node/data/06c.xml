<!DOCTYPE root SYSTEM "06c.dtd" [
<!ENTITY myname "Peter Jones">
]>
<root>&myname;</root>
