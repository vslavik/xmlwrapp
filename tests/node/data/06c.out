type_element
    type_entity_ref
        type_dtd_entity
            type_text
